OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -312,7 +312,13 @@ bool InverseKinematicsTool::run()
 
         const clock_t start = clock();
         double dt = 1.0/markersReference.getSamplingFrequency();
-        int Nframes = int((final_time-start_time)/dt)+1;
+        int Nframes = int((final_time-start_time)/dt + SimTK::SignificantReal)+1;
```

Comment:
Looks right, but how do you know only one `SimTK::SignificantReal` is necessary here? What is the precision of `markersReference.getSamplingFrequency()`? Are `start_time` and `final_time` both passed though the serializer (with precision 1e-6ish)? To avoid any chance of precision/roundoff issues, I usually do something like
```cpp
int Nframes = floor((final_time - start_time)/dt + 0.5) + 1;
```